server/libmain/libmain.go

```diff
@@ -62,7 +62,7 @@ var (
 	sslPort        = flag.Int("ssl_port", 8081, "The port to listen for HTTPS traffic on")
 	gRPCPort       = flag.Int("grpc_port", 1985, "The port to listen for gRPC traffic on")
 	gRPCSPort      = flag.Int("grpcs_port", 1986, "The port to listen for gRPCS traffic on")
-	monitoringPort = flag.Int("monitoring_port", 9090, "The port to listen for monitoring traffic on")
+	monitoringPort = flag.Int("monitoring_port", 9100, "The port to listen for monitoring traffic on")
```

Comment:
Can you also update the port elsewhere? (just grep for 9090 in both repos)

It's used a bunch in the deployment files.

Reply:
(You might also check any docs we have)

Reply:
N/A since I reverted this change